extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -44,12 +44,16 @@ amp-carousel[controls] .amp-carousel-button,
   left: 16px;
   background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="%23fff" viewBox="0 0 18 18"><path d="M15 8.25H5.87l4.19-4.19L9 3 3 9l6 6 1.06-1.06-4.19-4.19H15v-1.5z" /></svg>');
   background-size: 18px 18px;
+  opacity: 1;
+  visibility: visible;
```

Comment:
you can't do this as the behavior is intended. Please use the `controls` attribute of you want the controls to be permanently displayed.